packages/core/content-releases/admin/src/components/AddReleaseDialog.tsx

```diff
@@ -27,17 +35,36 @@ interface AddReleaseDialogProps {
 export const AddReleaseDialog = ({ handleClose }: AddReleaseDialogProps) => {
   const { formatMessage } = useIntl();
   const toggleNotification = useNotification();
+  const { push } = useHistory();
+  const { formatAPIError } = useAPIErrorHandler();
 
-  const handleSubmit = () => {
-    handleClose();
+  const [createRelease, { isLoading }] = useCreateReleaseMutation();
 
-    toggleNotification({
-      type: 'success',
-      message: formatMessage({
-        id: 'content-releases.modal.release-created-notification-success',
-        defaultMessage: 'Release created.',
-      }),
+  const handleSubmit = async (values: FormValues) => {
+    const release = await createRelease({
```

Comment:
This is completely personal, but for me this is not really the release, this is more the `response`.

So then you can do
```
if ('data' in response) {
  const { data } = response.data
  toggleNotification(.... /content-releases/${data.id} )
}
```

It's still not perfect so it's up to you!